developer/ruby/SubProjectClassGenerators/FileHeader.rb

```diff
@@ -3,32 +3,8 @@ def fileHeader
   fileHeader = String.new
 
   fileHeader << "/***********************************************************************************************************************\n"
-  fileHeader << "*  OpenStudio(R), Copyright (c) 2008-2023, Alliance for Sustainable Energy, LLC, and other contributors. All rights reserved.\n"
-  fileHeader << "*\n"
-  fileHeader << "*  Redistribution and use in source and binary forms, with or without modification, are permitted provided that the\n"
-  fileHeader << "*  following conditions are met:\n"
-  fileHeader << "*\n"
-  fileHeader << "*  (1) Redistributions of source code must retain the above copyright notice, this list of conditions and the following\n"
-  fileHeader << "*  disclaimer.\n"
-  fileHeader << "*\n"
-  fileHeader << "*  (2) Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following\n"
-  fileHeader << "*  disclaimer in the documentation and/or other materials provided with the distribution.\n"
-  fileHeader << "*\n"
-  fileHeader << "*  (3) Neither the name of the copyright holder nor the names of any contributors may be used to endorse or promote products\n"
-  fileHeader << "*  derived from this software without specific prior written permission from the respective party.\n"
-  fileHeader << "*\n"
-  fileHeader << "*  (4) Other than as required in clauses (1) and (2), distributions in any form of modifications or other derivative works\n"
-  fileHeader << "*  may not use the \"OpenStudio\" trademark, \"OS\", \"os\", or any other confusingly similar designation without specific prior\n"
-  fileHeader << "*  written permission from Alliance for Sustainable Energy, LLC.\n"
-  fileHeader << "*\n"
-  fileHeader << "*  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS \"AS IS\" AND ANY EXPRESS OR IMPLIED WARRANTIES,\n"
-  fileHeader << "*  INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE\n"
-  fileHeader << "*  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE UNITED STATES GOVERNMENT, OR THE UNITED\n"
-  fileHeader << "*  STATES DEPARTMENT OF ENERGY, NOR ANY OF THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,\n"
-  fileHeader << "*  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF\n"
-  fileHeader << "*  USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,\n"
-  fileHeader << "*  STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF\n"
-  fileHeader << "*  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.\n"
+  fileHeader << "*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.\n"
+  fileHeader << "*  See also https://openstudio.net/license\n"
```

Comment:
I realized this was outdated when I tried to use GenerateClass.rb to create a missing FT test for CoolingTowerSingleSpeed, so I decided to clean everything up